### PR TITLE
Migrating to Requests library as recommended in https://github.com/o1…

### PR DESCRIPTION
**Pull request by cstech-carl-camilleri**
…9s/solr-to-es/issues/17

### CODE REVIEW

**Comment by epugh** [comment]
@cstech-carl-camilleri does this let us remove the import pysolr?   I notice that is still in there.

File: solr_to_es/__main__.py [hunk 1]
```diff
@@ -69,15 +69,15 @@ def parse_args():
 
 
 def main():
-    try:
+    try:               
         args = parse_args()
         es_conn = Elasticsearch(hosts=args['elasticsearch_url'], timeout=args['es_timeout'])
 
         # Split the solr_url into the root and the request handler
         solr_conn = pysolr.Solr(args['solr_url'].rsplit('/', 1)[0], search_handler=args['solr_url'].rsplit('/', 1)[-1])
```
Comment (epugh) [inline solr_to_es/__main__.py:77]: Can we remove this?

File: solr_to_es/solrSource.py [hunk 2]
```diff
@@ -93,10 +103,13 @@ def __next__(self):
...
             if self.current * self.rows < self.max:
                 self.current += 1
-                response = self.solr_conn.search(self.query, rows=self.rows,
-                                                 start=(self.current - 1) * self.rows,
-                                                 **self.options)
-                self.docs = iter(response.docs)
+                url = self.solr_url+ '/select?q=' + self.query + '&rows=' + str(self.rows) + "&start=" + str(((self.current - 1) * self.rows))
+                if self.fl:
+                    url = url + "&fl=" + (','.join(self.fl))
+                print(url)
```
Comment (epugh) [inline solr_to_es/solrSource.py:109]: Should this still be here?

File: solr_to_es/solrSource.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import pysolr
```
Comment (epugh) [inline solr_to_es/solrSource.py:1]: can we remove this?

File: solr_to_es/__main__.py (hunk 1 above)
Comment (cstech-carl-camilleri) [reply solr_to_es/__main__.py:76]: thanks, yes pushed another commit to avoid this change

File: solr_to_es/solrSource.py (hunk 2 above)
Comment (cstech-carl-camilleri) [reply solr_to_es/solrSource.py:109]: Normally yes, any reason why we should remove building the URL to request?

File: solr_to_es/solrSource.py (hunk 3 above)
Comment (cstech-carl-camilleri) [reply solr_to_es/solrSource.py:1]: thanks, removed in the next commit

File: solr_to_es/solrSource.py (hunk 2 above)
Comment (epugh) [reply solr_to_es/solrSource.py:108]: I think I just meant the `print(url)`

**Comment by epugh** [comment]
This is looking really great!

File: solr_to_es/solrSource.py (hunk 2 above)
Comment (cstech-carl-camilleri) [reply solr_to_es/solrSource.py:108]: Sorry misunderstood that, `print(url)` has been removed in the last commit

**Pull request merged**